include/rocksdb/write_batch.h

```diff
@@ -181,6 +184,13 @@ class WriteBatch : public WriteBatchBase {
     // we won't break existing clients of Handler on a source code level when
     // adding a new member function.
 
+    // Default implementation will just call `PutCF()` without
+    // `KvProtectionInfo` for backwards compatibility.
+    virtual Status PutCF(uint32_t column_family_id, const Slice& key,
+                         const Slice& value, const KvProtectionInfo*) {
```

Comment:
I'm confused. This is a public interface and contains a class that is not defined in public. Do we expect users to use it?

Reply:
Hm, good point. We do not expect users to use it. However we need it in `MemTableInserter` which is used via a `Handler*`. I will think more how to achieve this without exposing the class name publicly... LMK any suggestion.

Reply:
Removed.